.github/workflows/ci.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14.x, 16.x, 18.x]
```

Comment:
does this not actually work in node 14 or are we just removing this because it is past LTS?

Reply:
Node 14 doesn't have fetch so cannot use Nano 11. The idea is Nano 11+ will be for Node 18 onwards. Folks needing Nano and still using 16 or older will have to use Node 10ish. Anyone for 10ish?